app/_how-tos/visualize-llm-metrics-with-grafana.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Visualize LLM traffic with Prometheus and Grafana"
+content_type: how_to
+related_resources:
+  - text: AI Gateway
+    url: /ai-gateway/
+  - text: AI Proxy Advanced
+    url: /plugins/ai-proxy-advanced/
+  - text: Prometheus plugin
+    url: /plugins/prometheus/
+  - text: Monitor AI metrics
+    url: /ai-gateway/monitor-ai-llm-metrics/
+
+description: Learn how to monitor LLM traffic and visualize AI metrics in Grafana using the AI Proxy Advanced and Prometheus plugins in {{ site.base_gateway }}.
+
+products:
+  - gateway
+  - ai-gateway
+
+works_on:
+  - on-prem
+
+min_version:
+  gateway: '3.11'
+
+plugins:
+  - ai-proxy-advanced
+  - prometheus
+
+entities:
+  - service
+  - route
+  - plugin
+
+tags:
+  - ai
+  - observability
+  - prometheus
+  - grafana
+
+tldr:
+  q: How can I visualize LLM traffic metrics in Kong AI Gateway?
+  a: |
+    Enable the AI Proxy Advanced plugin to collect detailed request and model statistics. Then configure the Prometheus plugin to expose these metrics for scraping. Finally, connect Grafana to visualize model performance, usage trends, and traffic distribution in real time.
+
+tools:
+  - deck
+
+prereqs:
+  inline:
```

Comment:
```suggestion
  konnect:
    - name: KONG_STATUS_LISTEN
      value: '0.0.0.0:8100'
  inline:
```
This one is required for Konnect with Prometheus. It listens for Prometheus on a different port that we have to specify when setting up the DP.